js/templates/listingCard.html

```diff
@@ -42,7 +42,10 @@
   </div>
   <% if (ob.vendor) { %>
   <a class="userIconWrapper js-userName">
-    <div class="userIcon disc clrBr2 clrSh1"
+    <% const vendorTip = `${ob.vendor.name}${ob.vendor.location ? `  📍  ${ob.vendor.location}` : ''}`;
```

Comment:
I think we should remove the location from displaying in the tooltip if that is the case.